conda/recipes/mantidworkbench/meta.yaml

```diff
@@ -44,7 +44,9 @@ requirements:
     - python.app  # [osx]
     - qtconsole {{ qtconsole }}
     - {{ pin_compatible("setuptools", max_pin="x.x") }}
+    {% if environ.get('INCLUDE_MANTIDDOCS', 'True') != 'False' %}
```

Comment:
I didn't even think of using environment variables to make this optional. This is a very clean way to accomplish the option.